presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/QueryBuilder.java

```diff
@@ -202,6 +197,24 @@ else if (typeAndValue.getType() instanceof CharType) {
         return statement;
     }
 
```

Comment:
to keep backwards compatibility, maybe you could add a function with the old parameters that calls this with ImmutableMap.of() as a default value eg
```suggestion
     public PreparedStatement buildSql(
            JdbcClient client,
            ConnectorSession session,
            Connection connection,
            String catalog,
            String schema,
            String table,
            List<JdbcColumnHandle> columns,
            TupleDomain<ColumnHandle> tupleDomain,
            Optional<JdbcExpression> additionalPredicate)
            throws SQLException
    {
    return buildSql(client, session, connection, catalog, schema, table, columns, ImmutableMap.of(), tupleDomain, additionalPredicate);
    }
```

Reply:
Yes, makes sense. I added it